### PR TITLE
Add exception in mesh and update the test of http request.

### PR DESCRIPTION
**Pull request by ifwx**

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/graphics/Mesh.java [hunk 1]
```diff
@@ -16,23 +16,10 @@
...
-import com.badlogic.gdx.graphics.glutils.IndexArray;
-import com.badlogic.gdx.graphics.glutils.IndexBufferObject;
-import com.badlogic.gdx.graphics.glutils.IndexBufferObjectSubData;
-import com.badlogic.gdx.graphics.glutils.IndexData;
-import com.badlogic.gdx.graphics.glutils.ShaderProgram;
-import com.badlogic.gdx.graphics.glutils.VertexArray;
-import com.badlogic.gdx.graphics.glutils.VertexBufferObject;
-import com.badlogic.gdx.graphics.glutils.VertexBufferObjectSubData;
-import com.badlogic.gdx.graphics.glutils.VertexData;
+import com.badlogic.gdx.graphics.glutils.*;
```
Comment (siondream) [inline gdx/src/com/badlogic/gdx/graphics/Mesh.java]: Why import everything? it wasn't hurting anybody.

**Comment by siondream** [comment]
What does the HttpRequestExample have to do with a shader being null? I mean it's nice the example is more elaborate but unsure as to why all of this comes together.

File: gdx/src/com/badlogic/gdx/graphics/Mesh.java (hunk 1 above)
Comment (ifwx) [inline gdx/src/com/badlogic/gdx/graphics/Mesh.java]: This is changed automatically by the compiler of Intellij IDEA 13.

**Comment by ifwx** [comment]
This submission contains three unrelated issues. Now, I have modified the title of the submission.

**Comment by xoppa** [comment]
Please create a separate pr for separate issues. As for the null check, I don't see why we would need that. If someone passes null to a method, then a npe is to be expected.

**Comment by ifwx** [comment]
Ok, I will create a separate problem. If ShaderProgram is null, it reports only NullPointer exception. Because the previous version does not require transfer ShaderProgram, NullPointer exception will mislead new hands.

**Comment by loogie** [comment]
Hello!

 

You have a new message, please read <http://impresoresomega.com/vitamin.php?vv>

 

loogie@basicelementdesign.com

**Pull request closed without merging**